src/shim/shim_syscall.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * The Shadow Simulator
+ * See LICENSE for licensing information
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <sys/syscall.h>
+#include <sys/time.h>
+#include <time.h>
+
+#include "shim/shim.h"
+#include "shim/shim_syscall.h"
+#include "support/logger/logger.h"
+
+#define SIMTIME_NANOS_PER_SEC 1000000000l
```

Comment:
Why the ending 1's? And should we move these constants to a header shared between shadow and the shim to ensure they stay sync'd?

Reply:
It's an `l` (a lower case `L`). I think I refactored this from a previously defined constant to this `#define`, but yeah we should use an existing definition or create one (e.g. in `definitions.h`).